classes/helper/Helper.php

```diff
@@ -117,7 +117,7 @@ public function generate()
     /**
      * @deprecated 1.5.0
      */
-    public static function renderAdminCategorieTree(
+    public static function renderAdminCategoriesTree(
```

Comment:
Hi  @developerarun 
Thanks for your contribution :)
I'm not sure it a good ideia to rename a public function, it can introduce Break changes !

Reply:
Agree with your concerns but that was deprecated and not called anywhere, so we can correct it for coming versions if possbile

Reply:
My be called in external modules for example :)